src/sources/prometheus/parser.rs

```diff
@@ -105,7 +105,7 @@ fn parse_tags(input: &str) -> Result<BTreeMap<String, String>, ParserError> {
             continue;
         }
         let pair = pair.trim();
-        let parts = pair.split('=').collect::<Vec<_>>();
+        let parts = pair.splitn(2, '=').collect::<Vec<_>>();
         if parts.len() != 2 {
```

Comment:
While this is the simplest change, this might be better expressed as the following to avoid creating a new Vec for what should always be exactly two elements:
```rust
match pair.find('=') {
    None => Err(…),
    Some(equals) => {
        let key = pair[..equals].trim().to_string();
        let mut value = pair[equals + 1..].trim();
        …
    }
```